.github/workflows/geocontrib.yml

```diff
@@ -0,0 +1,42 @@
+name: "Docker build"
+on:
+  push:
+  pull_request:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+    - name: "Checking out"
+      uses: actions/checkout@v1
+    - name: "Building docker image"
+      working-directory: docker
+      run: docker-compose -f docker-compose.geocontrib-alone.yaml build
+    - name: "Getting image tag"
+      id: version
+      run: echo ::set-output name=VERSION::$(echo $GITHUB_REF | cut -d / -f 3)
+    - name: "Logging in docker.io"
+      if: github.repository == 'neogeo-technologies/geocontrib'
+      uses: azure/docker-login@v1
+      with:
+        username: '${{ secrets.DOCKER_HUB_USERNAME }}'
+        password: '${{ secrets.DOCKER_HUB_PASSWORD }}'
+    - name: "Pushing latest to docker.io"
+      if: github.repository == 'neogeo-technologies/geocontrib' && github.event_name != 'release'
```

Comment:
No condition on branch name ? 
I think what matches the workflow here would be `develop` branch => `latest` tag, and `master` branch => `stable` tag.
Am I correct @bchartier ?